### ISSUE TITLE
Bar Chart not taking parent's full width (as it says on the docs)

### ISSUE THREAD

**Issue by henryornellas** [Abhinandan-Kushwaha/react-native-gifted-charts#660]
My code:
```
import { View } from "react-native";
import { BarChart } from "react-native-gifted-charts";
const stepsData = [
	{
		label: "Pendente",
		value: 140,
	},
	{
		label: "Em descarga",
		value: 62,
	},
	{
		label: "Presente",
		value: 154,
	},
	{
		label: "Concluído",
		value: 121,
	},
	{
		label: "No show",
		value: 152,
	},
];

export default function BarGraph() {
	return (
		<View className="border">
			<BarChart data={stepsData} />
		</View>
	);
}
```

Output:
![Sem título](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/106675861/fef062ca-0cbb-43e3-a22f-679f47876e84)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @henryornellas 
What is the issue here? Do you want the labels to not ellipsise?

**Comment by Abhinandan-Kushwaha** [comment]
Hi @henryornellas 👋
Thanks for requesting this feature. It is now available from version `1.4.11` onwards 🎉

Now you can pass the prop `adjustToWidth` to allow bars to occupy the screen width. You can also make them fit into the parent width by passing the prop `parentWidth` along with adjustToWidth.

Please use the latest version of the library.
